swagger/paging.json

```diff
@@ -173,7 +218,10 @@
     },
     "/paging/multiple/nextOperationWithQueryParams": {
       "get": {
-        "x-ms-pageable": { "nextLinkName": null, "itemName": "values" },
+        "x-ms-pageable": {
+          "nextLinkName": "null",

```

Comment:
@iscai-msft 

This is line wrong? Should it be `"nextLinkName": null` (no quotes)?

Reply:
oh yeah that does seem wrong. It's showing up in the diff because I reformatted the swagger, so this code has been wrong for a long time. Let me do a separate PR to update this test